build/Dockerfile

```diff
@@ -72,5 +74,6 @@ RUN chown -R steam:steam /usr/local/lib/dsta/ \
 EXPOSE 10999/udp
 VOLUME ["/var/lib/dsta/config"]
 
-WORKDIR /opt/dst/bin/
+WORKDIR /var/lib/dsta/config
```

Comment:
If we don't it and as we've already got the path in the `VOLUME`, we should remove it, I think.